Jenkinsfile.release

```diff
@@ -1,23 +1,31 @@
 #!/usr/bin/env groovy
 
 ansiColor('xterm') {
-  node('JenkinsMarathonCI-Debian8-2017-10-23') {
+  node('JenkinsMarathonCI-Debian8-2017-10-06') {
 
     properties([
       parameters([
-        string(name: 'gitsha',
-          defaultValue: '',
-          description: 'Git Commit SHA to Build and Release (ex. 167d7fb86 or 167)'
+        string(name: 'ref',
+          defaultValue: 'origin/releases/1.5',
```

Comment:
Shouldn't this be `releases/1.6`? Even if it does not exist.

Reply:
Actually, I think this should rather point to builds and not refs. The ref is inferred from the builds